src/analysis/validation.jl

```diff
@@ -5,11 +5,24 @@ Perform EPANET hydraulic simulation (via WNTR) and compute timeseries of flows a
 
 using DataFrames
 
+
+function _get_wntr_node_attribute(wntr_simulation, node_name::String, attribute::String)
```

Comment:
I got this when I tested your branch:
```
ERROR: LoadError: MethodError: no method matching _get_wntr_node_attribute(::PyCall.PyObject, ::SubString{String}, ::String)
Closest candidates are:
  _get_wntr_node_attribute(::Any, ::String, ::String) at /Users/jstickel/.julia/dev/WaterModelsAnalytics/src/analysis/validation.jl:9
```
changing `node_name::String` to `node_name::AbstractString` resolved the error.